neqo-transport/src/connection/tests/mod.rs

```diff
@@ -323,3 +327,19 @@ fn split_datagram(d: &Datagram) -> (Datagram, Option<Datagram>) {
         b.map(|b| Datagram::new(d.source(), d.destination(), b)),
     )
 }
+
+fn get_tokens(client: &mut Connection, now: Option<Instant>) -> Vec<ResumptionToken> {
+    if let Some(t) = now {
+        let _ = client.process_output(t);
+    }
```

Comment:
Could we make the Instant argument mandatory AND check that process_output does not produce a datagram?  I'm thinking that this is almost always going to generate an ACK: receiving a session ticket will often be the first packet received by a client after HANDSHAKE_DONE.  connect_force_idle forces that to be acknowledged, but if we aren't using that, the client will want to send an ACK.

If we can't ensure that there is no packet generated, we should log any that are dropped as a result so that there are no surprises.